README.rst

```diff
@@ -42,12 +42,13 @@ You may also pass additional parameters.
 
 .. code:: python
 
-    monitoring_thread = start_monitoring(seconds_frozen=10, tests_per_second=10)
+    monitoring_thread = start_monitoring(seconds_frozen=10, test_interval=100)
 
 The values in the example are defaults. This mean the check will happen 10
 times per second. If a thread is frozen for at least 10 seconds then the stack
```

Comment:
> This mean the check will happen 10 times per second.

This text is about `tests_per_second` arg. What if we change this with something like "check will happen every 100 milliseconds"?